e0works/MyDiary-front
# #161938353 user should be able to view list of dairy entry

#### What does this PR do?
- implement list diary entries

#### Description of Task to be completed?
- add list entry action
- create a single item component
- create a single item helper

#### How should this be manually tested?
- start the app
- log in and create a new diary entry
- you should see your new diary entry on the list

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#161938353](https://www.pivotaltracker.com/story/show/161938353, "view list diary entries")

#### Screenshots (if appropriate)

<img width="1680" alt="screen shot 2018-11-30 at 6 44 55 pm" src="https://user-images.githubusercontent.com/24890605/49307105-7f40d700-f4d4-11e8-8494-7a32d9f24fcd.png">


#### Questions:
